nixos/modules/services/misc/gitea.nix

```diff
@@ -454,6 +471,41 @@ in
       ];
     };
 
+    services.nginx = lib.mkIf cfg.configureNginx {
+      enable = true;
+      virtualHosts."${cfg.domain}" = {
+        root = "/var/empty";
+
+        locations."/" = {
+          proxyPass = (if cfg.enableUnixSocket then "http://unix:/run/gitea/gitea.sock" else "http://127.0.0.1:${toString(cfg.httpPort)}");
+          recommendedProxySettings = true;
+        };
+
+        locations."/avatars/" = {
```

Comment:
Shouldn't this use the ``~ ^`` to be regex proof?

Reply:
Replaced to `^~`.